rust/src/worker.rs

```diff
@@ -404,6 +410,10 @@ impl Inner {
             ));
         }
 
+        if let Some(disable_liburing) = disable_liburing {
```

Comment:
I assume here we are checking if user given `disable_liburing` is true.

Reply:
No, you're checking that `disable_liburing` is `Some()`, for checking what is inside you'd also have to check the boolean `disable_liburing` itself. Can be simplified to `disable_liburing == Some(true)`. One more reason to not have tri-state boolean though.